sdk/loadtestservice/azure-developer-loadtesting/swagger/README.md

```diff
@@ -0,0 +1,20 @@
+## Generate autorest code
+
+```yaml
+require: https://github.com/Azure/azure-rest-api-specs/blob/9401446f22177696d920cf110893a0de7452ee9e/specification/loadtestservice/data-plane/readme.md
+output-folder: ../
+java: true
+regenerate-pom: false
+title: LoadTestClient
+security: AADToken
+security-scopes: https://loadtest.azure-dev.com/.default
+data-plane: true
+generate-tests: true
+generate-models: true
```

Comment:
Not sure if you want to start this right now. This is only as reference, when you want to write convenient API and need to see what model would look like (but it might not what you want to use in API).

Reply:
@weidongxu-microsoft @samvaity The PR is updated to be non-DPG. No API is officially long-running. There are a few APIs which internally functions like long-running but our service doesn't treat it as one

Reply:
Yes, data-plane: true is not passed now, and pageable is added for required APIs

Reply:
@weidongxu-microsoft @samvaity  in case we want to use DPG, would the users be expected to use BinaryData to interact with SDK? Or would models still need to be created? In case of DPG, is it fine to use partial update with models in place of BinaryData in protocol methods?

Reply:
@weidongxu-microsoft Since multipart is not supported, is it fine to perform multipart upload manually as demonstrated in this commit?
https://github.com/Azure/azure-sdk-for-java/commit/7fcfb51efce5cd28602fd9a048821bdd426fd62d

Reply:
If release as DPG, user is expected to use BinaryData to interact with backend.

The point of DPG is that service can add model and convenience API (via partial update), after SDK released a few versions (even after a GA version).
Service is not required to have model or convenience API for preview or even for GA (though GA require much more test cases).

---

Yes, if payload is not large, one can use this approach to send multipart (I think I've tried that and it works, though likely one year ago).